docs/self-managed/operational-guides/update-guide/870-to-880.md

```diff
@@ -6,6 +6,140 @@ description: "Review which adjustments must be made to migrate from Camunda 8.7.
+- `operate-import-position-8.3.0_`
+
+If all entries show the `completed` property set to `true`, the importing is done.
+
+Alternatively, metrics can be validated via:
+
+- `operate_import_completed`
+- `tasklist_import_completed`
+
+These properties can be used in the following example query:
+
+```
+sum(tasklist_import_completed{namespace="$namespace"}) by (importPositionAlias)
+```
+
+![metrics-import-complete](./img/import-complete-metrics.png)
+
+#### Update any custom prefixes
+
+:::important
+This step is required if you use an installation with custom prefixes.
+:::
+
+The new harmonized index schema introduces one common index prefix. To update to Camunda 8.8, indices must first migrate to one common index, so the Camunda Exporter, REST API, and other consumers can be configured correctly.
+
+##### Prefix migration
+
+:::warning
+Disk usage will massively increase while the migration is ongoing (and will return to normal when complete).
+Ensure you have at least double the disk space free.
+:::
+
+:::note
+While we test and validate the provided migration script,
+it is recommended that you first back up the database, and test the migration script in a test environment first
+to ensure there are no unexpected issues.
+:::
+
+:::note
+The prefix migration involves downtime, as all data in the secondary storage needs to be migrated to a different prefix, and writes need to be stopped in between.
+:::
+
+Complete the following steps to migrate any custom prefixes:
+
+1. Shut down your Camunda 8.7 cluster (for example, reduce the replicas to zero). No traffic can be executed against your to-be-migrated ES/OS prefixes.
+2. In the `camunda/bin` directory there is a `prefix-migration` script that will execute the prefix migration, which can be executed locally.
+   Pass the desired prefix using the `CAMUNDA_DATABASE_INDEXPREFIX` environment variable.
+
+   1. Make sure that your configuration in `camunda/config` has set the previous custom prefixes. Such that the prefix migration script can find them.
+   2. Alternatively, you can set them via env vars.
+
+   ```shell
+   export CAMUNDA_OPERATE_ELASTICSEARCH_INDEX_PREFIX=old-operate-prefix; \
+   export CAMUNDA_TASKLIST_ELASTICSEARCH_INDEX_PREFIX=old-tasklist-prefix; \
+   export CAMUNDA_DATABASE_INDEXPREFIX=some-new-prefix; \
+   ./prefix-migration
+   ```
+
+With the migration finished, the following configuration changes must be made prior to updating to 8.8.
+
+:::note
+Without these configuration changes, the update to 8.8 will fail.
+:::
+
+Given a new desired prefix of `some-new-prefix`:
+
+- The configured `CAMUNDA_OPERATE_ELASTICSEARCH_INDEX_PREFIX` must now be `some-new-prefix`.
+- The configured `CAMUNDA_TASKLIST_ELASTICSEARCH_INDEX_PREFIX` must now be `some-new-prefix`.
+- The prefix for the [Camunda exporter](/self-managed/zeebe-deployment/exporters/camunda-exporter.md) should be configured using `ZEEBE_BROKER_EXPORTERS_CAMUNDA_ARGS_INDEX_PREFIX` to `some-new-prefix`.
```

Comment:
This section says configuration changes must be made, but these don't feel like instructions. If the user _must_ set this config following the completion of the script, I'd recommend the following: 
```suggestion
Given a new desired prefix of `some-new-prefix`:

- Set `CAMUNDA_OPERATE_ELASTICSEARCH_INDEX_PREFIX` to `some-new-prefix`.
- Set `CAMUNDA_TASKLIST_ELASTICSEARCH_INDEX_PREFIX` to `some-new-prefix`.
- Set `ZEEBE_BROKER_EXPORTERS_CAMUNDA_ARGS_INDEX_PREFIX` (the prefix for the new [Camunda exporter](/self-managed/zeebe-deployment/exporters/camunda-exporter.md) to `some-new-prefix`.
```